tests/integrationTests/inputs/geom/slabBurner.v1.stp

```diff
@@ -0,0 +1,289 @@
+ISO-10303-21;
```

Comment:
I don't think we are using the slabBurner geom in the repo, can we remove this file?